routes/userProfile.js

```diff
@@ -83,4 +83,24 @@ router.post('/profile/update', authorization, async (req, res) => {
   }
 })
 
+// return searched users from search bar
+router.get('/search/', authorization, async (req, res) => {
+  try {
+    const { searchParam } = req.query
+    console.log(searchParam.toUpperCase())
+    const userSearchList = await db.query(
+      "SELECT user_id, first_name, last_name FROM user_profile WHERE UPPER(CONCAT(first_name, ' ', last_name)) LIKE $1",
+      [`%${searchParam.toUpperCase()}%`]
+    )
+    return res.json(userSearchList.rows)
+  } catch (err) {
+    console.error(err.message)
+    return res
+      .status(500)
+      .json(
+        `Server Error. Could not search users with parameter ${searchParam}`
+      )
```

Comment:
500, is a server error, but it's not the server's fault theres not a user with that name. You want a 404 or something to match your error message

Reply:
seems to be a confusion between the status and a message